cypress/support/fragments/exportManager/exportManagerSearchPane.js

```diff
@@ -0,0 +1,107 @@
+import { Pane, Button, TextField, MultiColumnListCell, Accordion, Checkbox, Modal } from "../../../../interactors";
+import { including } from "bigtest";
+
+const searchButton = Button({ type: 'submit' });
+const userSearchResults = Pane('User Search Results')
+const getSearchResult = (row = 0, col = 0) => MultiColumnListCell({ 'row': row, 'columnIndex': col });
+
+export default {
+	getSearchResult,
+	waitLoading() {
+		cy.expect(Pane('Export jobs').exists());
+	},
+	searchById(id) {
+		cy.do([
+			TextField().fillIn(id),
+			searchButton.click(),
+		])
+	},
+	selectSearchResultItem(indexRow = 0) {
+		return cy.do(this.getSearchResult(indexRow, 0).click());
+	},
+	closeExportJobPane() {
+		cy.do(Button({ ariaLabel: 'Close Export job ' }).click());
+	},
+	resetAll() {
+		cy.do(Button('Reset all').click());
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+	},
+	resetJobType() {
+		cy.do(Accordion({ id: 'type' }).find(Button({ icon: 'times-circle-solid' })).click());
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+	},
+	searchByScheduled() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-scheduled' })).click());
+	},
+	searchByInProgress() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-in-progress' })).click());
+	},
+	searchBySuccessful() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-successful' })).click());
+	},
+	searchByFailed() {
+		//Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
+		cy.do(Accordion({ id: 'status' }).find(Checkbox({ id: 'clickable-filter-status-failed' })).click());
```

Comment:
move to const Accordion({ id: 'status' })

Reply:
done